install-crossplane.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Does this script suppose to be identical to the one that is provided as a configmap?
If so, I think we should find a way to only have one copy (i.e. either find a way to use the configmap where this file is needed or mount this file instead of the configmap).

Reply:
Yes, this script is similar to configmap and installation with script is working. I just want to keep the configmap but the pod is failing so added both, once I resolve the issue with configmap will remove the script.